### DISCUSSION TITLE
issues模板是咋搞得

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#3574] [讨论]
[这个](https://github.com/Hex-Dragon/PCL2/issues/new/choose)是咋搞得，把docs翻遍了，没搞懂

**Comment by MoYuan-CN** [comment]
参考文档：[为仓库配置议题模板：配置模板选择器](https://docs.github.com/zh/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser)

你只需要配置那些后面按钮为 `Open` 的（即链接）就行
对于按钮为 `Get started` 的，会自动根据其对应的 Issue Form YML 文件内的配置显示

**Reply by MoYuan-CN** [reply]
哦对了，这个仓库的 Issue Template 用的是 YML 配置的 Form，而不是 Markdown 配置的模板
注意对应文档：[为仓库配置议题模板：创建议题表单](https://docs.github.com/zh/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms)

**Comment by Deep-Dark-Forest** [comment]
你也可以在仓库中创建一个`.github`文件夹，然后在`.github`文件夹中创建一个子文件夹，名称为`ISSUE_TEMPLATE`，然后在`ISSUE_TEMPLATE`中创建一个`.yml`文件，在文件中输入一些东西就行了

**Comment by wuliaodexiaoluo** [comment]
在 项目的 .github文件夹下新建 ISSUE_TEMPLATE，写一个yml配置文件
